includes/class-wc-payments-account.php

```diff
@@ -643,4 +648,29 @@ private function settings_changed( $changes = [] ) {
 		$diff = array_diff_assoc( $changes, $account );
 		return ! empty( $diff );
 	}
+
+	/**
+	 * Check if the named note has been actioned.
+	 *
+	 * @param string $note_name The name of the note to check.
+	 *
+	 * @return bool Flag indicating whether the note has been actioned.
+	 */
+	private function note_actioned( string $note_name ) {
+		// TODO: Can we look-up by name?
```

Comment:
Can use `get_notes_with_name` from the data store, as seen in [this post](https://developer.woocommerce.com/2020/10/16/using-the-admin-notes-inbox-in-woocommerce/).

Reply:
Ah ha, great. Thanks!